source/fundamentals/connection/server-selection.txt

```diff
@@ -66,47 +67,38 @@ selector function:
    ...     return servers
    ...
 
-
-
-Finally, we can create a ``~pymongo.MongoClient`` instance with this
-server selector.
-
+Finally, create a ``~pymongo.MongoClient`` instance with the
+server selector:
 
 .. code-block:: python
 
    >>> client = MongoClient(server_selector=server_selector)
 
-
-
 Server Selection Process
 ------------------------
 
-This section dives deeper into the server selection process for reads and
-writes. In the case of a write, the driver performs the following operations
-(in order) during the selection process:
-
-
-#. Select all writeable servers from the list of known hosts. For a replica set
-   this is the primary, while for a sharded cluster this is all the known mongoses.
-
-#. Apply the user-defined server selector function. Note that the custom server
-   selector is **not** called if there are no servers left from the previous
-   filtering stage.
-
-#. Apply the ``localThresholdMS`` setting to the list of remaining hosts. This
-   whittles the host list down to only contain servers whose latency is at most
-   ``localThresholdMS`` milliseconds higher than the lowest observed latency.
+This section describes the server selection process for reads and
+writes. For writes, the driver performs the following operations, in order,
+during the selection process:
 
-#. Select a server at random from the remaining host list. The desired
-   operation is then performed against the selected server.
+- Select all writeable servers from the list of known hosts. For a replica set,
```

Comment:
Since order matter, this should be a numbered list
```suggestion
1. Select all writeable servers from the list of known hosts. For a replica set,
```